plugins/fossa/src/components/FossaCard/FossaCard.tsx

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  EmptyState,
+  InfoCard,
+  MissingAnnotationEmptyState,
+  Progress,
+  useApi,
+} from '@backstage/core';
+import { useAsync } from 'react-use';
+import { Entity } from '@backstage/catalog-model';
+import { fossaApiRef } from '../../api';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  FOSSA_PROJECT_NAME_ANNOTATION,
+  useProjectName,
+} from '../useProjectName';
+import { Grid, Tooltip } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  numberError: {
+    fontSize: '5rem',
+    textAlign: 'center',
+    fontWeight: theme.typography.fontWeightMedium,
+    margin: theme.spacing(2, 0),
+    color: theme.palette.error.main,
+  },
+  numberSuccess: {
+    fontSize: '5rem',
+    textAlign: 'center',
+    fontWeight: theme.typography.fontWeightMedium,
+    margin: theme.spacing(2, 0),
+    color: theme.palette.success.main,
+  },
+  description: {
+    fontSize: '1rem',
+    textAlign: 'center',
+    fontWeight: theme.typography.fontWeightMedium,
+    color: theme.palette.text.secondary,
+  },
+  disabled: {
+    backgroundColor: theme.palette.background.default,
+  },
+  lastAnalyzed: {
+    color: theme.palette.text.secondary,
+    textAlign: 'center',
+  },
+  branch: {
+    textDecoration: 'underline dotted',
+  },
+}));
+
+export const FossaCard = ({
+  entity,
+  variant = 'gridItem',
+}: {
+  entity: Entity;
+  variant?: string;
+}) => {
+  const fossaApi = useApi(fossaApiRef);
+
+  const projectTitle = useProjectName(entity);
```

Comment:
Bit confusing naming here since `useProjectName` isn't a hook, could be `getProjectName` or so